src/Sulu/Component/Hash/Serializer/Subscriber/HashSerializeEventSubscriber.php

```diff
@@ -13,7 +13,7 @@
 
 use JMS\Serializer\EventDispatcher\EventSubscriberInterface;
 use JMS\Serializer\EventDispatcher\ObjectEvent;
-use JMS\Serializer\GenericSerializationVisitor;
+use JMS\Serializer\JsonSerializationVisitor;
```

Comment:
Is that change required? This means this subscriber is only working for JSON serialization...

Reply:
currently not required but required when we will upgrade to 2.0